.github/actions/prepare-build-env/action.yml

```diff
@@ -1,5 +1,10 @@
 name: Prepare build environment (Linux only)
 description: Set up Java, Maven, Gradle, etc.
+inputs:
```

Comment:
I don't remember why I added this, but it appears to be no longer used (or useful).